### ISSUE TITLE
Videomaker: Add arrows to Post Navigation

### ISSUE THREAD

**Issue by beafialho** [Automattic/themes#4893]
Can we add arrows to the post navigation, as shown in the example below?

**Demo Site**
<img width="188" alt="Captura de ecrã 2021-10-21, às 15 31 29" src="https://user-images.githubusercontent.com/65220155/138314790-f9603171-11e5-4029-8820-cb862e08fa5e.png">

**Figma**
<img width="583" alt="Captura de ecrã 2021-10-21, às 15 31 15" src="https://user-images.githubusercontent.com/65220155/138314797-1684b8d8-56ef-471a-89bf-4e3ac13c7f70.png">

**Comment by danieldudzic** [comment]
@scruffian Should I just use https://github.com/Automattic/themes/pull/4909/files here?

**Comment by scruffian** [comment]
I don't think it will work - we need this issue to be fixed first: https://github.com/WordPress/gutenberg/issues/35954

I think we should move this to blocked for now.

**Comment by alaczek** [comment]
Merged the fixed in the premium repo - closing as fixed.
